UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogItem.cs

```diff
@@ -148,5 +152,10 @@ public override string ToString()
 		{
 			return logEntry.ToString();
 		}
+
+		private void CopyLogText()
+		{
+			GUIUtility.systemCopyBuffer = logEntry.ToString();
```

Comment:
is this tested? we use ``TextUtils.CopyTextToClipboard(stringHere)`` to copy to clipboard when doing the ``copyid`` command. I think we had troubles with other methods of copying to clipboard.

Reply:
I tested it and it works for me.

Reply:
tested on a build? I can build it tomorrow and see myself, kinda late for me now.

Reply:
> tested on a build? I can build it tomorrow and see myself, kinda late for me now.

Will build the game in a few minutes and test it.

Reply:
tested. works fine.